README.Rmd

```diff
@@ -69,14 +69,14 @@ pak::pak("duckdblabs/duckplyr")
 
 ```{r attach}
 library(conflicted)
-library(duckplyr)
+library(dplyr)
 conflict_prefer("filter", "duckplyr")
```

Comment:
do you still want this particular conflict_prefer call?

Reply:
```suggestion
conflict_prefer("filter", "dplyr")
```

Reply:
Good catch, thanks!